Examples/ModalExample/ModalExample/AppDelegate.swift

```diff
@@ -21,7 +21,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Default to demo production account (safe to share the API publicly)
         let apiKey = UserDefaults.standard.string(forKey: ApiKeyKey) ?? "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjb21wYW55X2lkIjoiNWRmZTM2OGEwOWI2ZWYwMDAxYjNlNjE4IiwiaWF0IjoxNTc2OTQxMTk0fQ.QBWr2goMwOngVhi6wY9sdFAKEvBGmn-JRDKstVMFh6M"
         let apiHost = Environment(rawValue: UserDefaults.standard.integer(forKey: EnvironmentKey)) ?? Environment.Production == Environment.Development ? EnvironmentUrl.Development : EnvironmentUrl.Production
-        Iterate.shared.configure(apiKey: apiKey, apiHost: apiHost.rawValue, surveyTextFontName: "Merriweather-Regular", buttonFontName: "WorkSans-Regular")
+        Iterate.shared.configure(apiKey: apiKey, apiHost: apiHost.rawValue, surveyTextFontName: "Georgia", buttonFontName: "WorkSans-Regular")
```

Comment:
Was this intentional? Ideally we'd leave Merriweather to test custom fonts

Reply:
Ah yea, I did change this because this custom font just specifies the "Regular" type...so bolding and italic wasn't working, but it works with you pass a whole font family. I'll revert though for testing purposes